Update configuration-settings.rst (#5270)
mattermost/docs: source/messaging/team-settings.rst

Allow only users with a specific email domain to join this team
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Specifying domains in this setting prevents users from joining the team when their email domain is not on the list. Only users that have an email domain matching the defined domains may join the team. 

Reviewer: ```suggestion
Specifying domains in this setting prevents users from joining the team when their email domain is not on the list. Only users that have an email domain matching the defined domains may join the team.
```

> Users without a matching domain on the team prior to the domain being specified will not be removed after the domain is added. For domains to be restricted, **System Console > Authentication > Email > Require Email Verification** must be set to ``True``. 

Reviewer 2: ```suggestion
Users without a matching domain on the team prior to the domain being specified will not be removed after the domain is added. For domains to be restricted, **System Console > Authentication > Email > Require Email Verification** must be set to ``true``. 
```

> Users without a matching domain on the team prior to the domain being specified will not be removed after the domain is added. For domains to be restricted, **System Console > Authentication > Email > Require Email Verification** must be set to ``true``. 

Reviewer 3: ```suggestion
Users without a matching domain on the team prior to the domain being specified will not be removed after the domain is added. For servers with email authentication **System Console > Authentication > Email > Require Email Verification** must be set to ``true`` for domain restrictions to be effective.
```

> Users without a matching domain on the team prior to the domain being specified will not be removed after the domain is added. For servers with email authentication **System Console > Authentication > Email > Require Email Verification** must be set to ``true`` for domain restrictions to be effective.

Reviewer: ```suggestion
Users without a matching domain on the team prior to the domain being specified will not be removed after the domain is added. For servers with email authentication **System Console > Authentication > Email > Require Email Verification** must be set to **true** for domain restrictions to be effective.
```
